crates/matrix-sdk-base/src/rooms/normal.rs

```diff
@@ -69,13 +70,27 @@ use crate::{
     MinimalStateEvent, OriginalMinimalStateEvent, RoomMemberships,
 };
 
+/// Each time a room's RoomInfo is changed, the roominfo_update_receiver in the
+/// BaseClient will receive this struct containing the update information.
+/// The sender must decide if this update should cause a room list update.
+/// If the sender knows that another action has already caused the room to
+/// update, it should not send another update.
```

Comment:
This comment is too specific about struct names and field names. Let's make it more generic too, by saying that it describes an update to a `RoomInfo`, with extra data for updates caused by sliding sync that indicates whether an update may be spurious or not.